.travis.yml

```diff
@@ -22,7 +23,8 @@ env:
 
 matrix:
   include:
-    - env: TOXENV=translations SQLALCHEMY_DATABASE_URI="$SQLALCHEMY_DATABASE_TEST_URI" PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/ePROMs-site-config/develop/site_persistence_file.json"
```

Comment:
After emailing with Tyler it seems I was using a project-specific (TrueNTH) access token I had generated instead of an account-specific token that should work across projects.

It seems to be now working after plugging in the new token Tyler gave us (still waiting on tests to finish :) )